evennia/utils/evform.py

```diff
@@ -61,8 +61,8 @@
 ```python
 from evennia import EvForm, EvTable
 
-# create a new form from the template
-form = EvForm("path/to/testform.py")
+# create a new form from the template - do not include the .py of the filename!
+form = EvForm("path/to/testform")
```

Comment:
I don't think this correction is, well, correct. 

The module parser _does_ require `.py` if the path is given as a full unix-style path, like exemplified here: `path/to/testform.py` (e.g. `/home/foo/mygame/world/testform.py`) 

However, if using a _python path_ (which is also allowed), you mustn't use the .py suffix, like `path.to.testform` (e.g. `world.testform`). 

I would suggest changing this PR to instead provide the python path (`path.to.testform`) since it's probably more likely to be used than the unix path.

Reply:
Thanks, I wasn't aware that you could use either method.